pkg/certificate/identity.go

```diff
@@ -370,25 +370,15 @@ func PrepareIdentityCsrOptions(idCfg *config.IdentityConfig, domain, service str
 		return nil, err
 	}
 
+	// TODO: deprecate: ATHENZ_SIA_DEFAULT_COUNTRY, ATHENZ_SIA_DEFAULT_PROVINCE, ATHENZ_SIA_DEFAULT_ORGANIZATION, ATHENZ_SIA_DEFAULT_ORGANIZATIONAL_UNIT
+	// TODO: use DEFAULT_SUBJECT as default values
 	subject := pkix.Name{
-		Country: func() []string {
-			if config.DEFAULT_COUNTRY != "" {
-				return []string{config.DEFAULT_COUNTRY}
-			}
-			return nil
-		}(),
-		Province: func() []string {
-			if config.DEFAULT_PROVINCE != "" {
-				return []string{config.DEFAULT_PROVINCE}
-			}
-			return nil
-		}(),
-		Organization: func() []string {
-			if config.DEFAULT_ORGANIZATION != "" {
-				return []string{config.DEFAULT_ORGANIZATION}
-			}
-			return nil
-		}(),
+		Country:            idCfg.ServiceCert.CopperArgos.Subject.Country,
+		Province:           idCfg.ServiceCert.CopperArgos.Subject.Province,
+		Organization:       idCfg.ServiceCert.CopperArgos.Subject.Organization,
+		Locality:           idCfg.ServiceCert.CopperArgos.Subject.Locality,
+		StreetAddress:      idCfg.ServiceCert.CopperArgos.Subject.StreetAddress,
```

Comment:
(IF POSSIBLE only; didn't test it)
```suggestion
subject := idCfg.ServiceCert.CopperArgos.Subject
// done
```

Reply:
(again if you think its a good idea but wanna make a change in separate pr, writing todo and move on is fine for sure)

Reply:
fixed in 383b6c3